examples/platform/silabs/SiWx917/SiWx917/sl_wifi_if.c

```diff
@@ -655,14 +721,169 @@ static sl_status_t wfx_rsi_do_join(void)
                 wfx_retry_interval_handler(is_wifi_disconnection_event, wfx_rsi.join_retries);
                 if (is_wifi_disconnection_event || wfx_rsi.join_retries <= MAX_JOIN_RETRIES_COUNT)
                 {
-                    xEventGroupSetBits(wfx_rsi.events, WFX_EVT_STA_START_JOIN);
+                    event.eventType = WFX_EVT_STA_START_JOIN;
+                    WfxPostEvent(&event);
                 }
             }
         }
     }
     return status;
 }
 
+void HandleDHCPPolling()
+{
+    struct netif * sta_netif;
+    WfxEvent_t event;
+
+    sta_netif = wfx_get_netif(SL_WFX_STA_INTERFACE);
```

Comment:
add a null check before use

Reply:
Added check, followup on how to properly handle.